tests/ha/test_multi_server_pod_failure.py

```diff
@@ -0,0 +1,287 @@
+                    deployment_backup = None
+                else:
+                    deployment_name = self.pod_dict.get(pod_name)[2]
+                    deployment_backup = self.pod_dict.get(pod_name)[1]
+                resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                               restore_method=self.restore_method,
+                                               restore_params={"deployment_name": deployment_name,
+                                                               "deployment_backup":
+                                                                   deployment_backup})
+                LOGGER.debug("Response: %s", resp)
+                assert_utils.assert_true(resp[0], "Failed to restore pod by "
+                                                  "{self.restore_method} way")
+                LOGGER.info("Successfully restored pod %s by %s way",
+                            pod_name, self.restore_method)
+        if os.path.exists(self.test_dir_path):
+            sysutils.remove_dirs(self.test_dir_path)
+        # TODO: As cluster restart is not supported until F22A, Need to redeploy cluster after
+        #  every test
+        if self.deploy:
+            LOGGER.info("Cleanup: Destroying the cluster ")
+            resp = self.deploy_lc_obj.destroy_setup(self.node_master_list[0],
+                                                    self.node_worker_list,
+                                                    const.K8S_SCRIPTS_PATH)
+            assert_utils.assert_true(resp[0], resp[1])
+            LOGGER.info("Cleanup: Cluster destroyed successfully")
+
+            LOGGER.info("Cleanup: Setting prerequisite")
+            self.deploy_lc_obj.execute_prereq_cortx(self.node_master_list[0],
+                                                    const.K8S_SCRIPTS_PATH,
+                                                    const.K8S_PRE_DISK)
+            for node in self.node_worker_list:
+                self.deploy_lc_obj.execute_prereq_cortx(node, const.K8S_SCRIPTS_PATH,
+                                                        const.K8S_PRE_DISK)
+            LOGGER.info("Cleanup: Prerequisite set successfully")
+
+            LOGGER.info("Cleanup: Deploying the Cluster")
+            resp_cls = self.deploy_lc_obj.deploy_cluster(self.node_master_list[0],
+                                                         const.K8S_SCRIPTS_PATH)
+            assert_utils.assert_true(resp_cls[0], resp_cls[1])
+            LOGGER.info("Cleanup: Cluster deployment successfully")
+
+        LOGGER.info("Cleanup: Check cluster status")
+        resp = self.ha_obj.poll_cluster_status(self.node_master_list[0], timeout=180)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Cleanup: Cluster status checked successfully")
+
+        LOGGER.info("Done: Teardown completed.")
+
+    # pylint: disable=too-many-statements
```

Comment:
Is this needed for this test?

Reply:
Yes this is required, because after removing this pylint is giving error "tests/ha/test_multi_server_pod_failure.py:207:4: R0915: Too many statements (54/50) (too-many-statements)
"